### PR TITLE
DEMO-102: Nova SEO bundle in Studio Demo Site

### PR DESCRIPTION
**Pull request by damianz5**
https://jira.ez.no/browse/DEMO-102

related issue: https://github.com/ezsystems/ezstudio-demo-bundle/pull/185

### CODE REVIEW

File: doc/apache2/Readme.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Example config for Apache 2.4 in prefork mode:
         # Makes it possible to place your favicon at the root `web` directory in your eZ instance.
         # It will then be served directly.
         RewriteRule ^/favicon\.ico - [L]
-        RewriteRule ^/robots\.txt - [L]
+        # RewriteRule ^/robots\.txt - [L]
```
Comment (SylvainGuittard) [inline doc/apache2/Readme.md]: I think we'll have some merge conflicts during the upgrades if we comment this line. 
Do we have a better option?

File: doc/apache2/Readme.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ If you do not have an access to use virtualhost config, use the `.htaccess` file
 
     # Makes it possible to placed your favicon and robots.txt at the root of your web folder
     RewriteRule ^favicon\.ico - [L]
-    RewriteRule ^robots\.txt - [L]
+    # RewriteRule ^robots\.txt - [L]
```
Comment (SylvainGuittard) [inline doc/apache2/Readme.md]: Same as above.

File: app/config/config.yml [hunk 3]
```diff
@@ -32,6 +32,35 @@ ez_search_engine_solr:
...
+                    href: { location_id: 2 }
+                    title: 'Home'
+                Search:
+                    href: { route: 'ezpublish_ezstudiodemo_search' }
+                    title: 'Search'
+                'Shortcut icon':
+                    href: { asset: '/bundles/ezstudiodemo/images/favicon.ico' }
+                    type: 'image/x-icon'
+            default_metas:
+                author: "eZ Systems"
```
Comment (kamilmusial) [inline app/config/config.yml]: Is there a reason to mix single and double quotes?

File: doc/apache2/Readme.md (hunk 1 above)
Comment (damianz5) [inline doc/apache2/Readme.md]: It was suggested by the authors ([link](https://github.com/Novactive/NovaeZSEOBundle/blob/master/Resources/doc/INSTALL.md#remove-the-robotstxt-native-rewriterules-)), but if there will be no entries for `robots_disallow` in config.yml file - this change can be removed.

File: app/config/config.yml (hunk 3 above)
Comment (damianz5) [inline app/config/config.yml]: changed

**Comment by damianz5** [comment]
@kamilmusial @SylvainGuittard final review please

**Pull request merged**